src/api/app/models/workflow.rb

```diff
@@ -21,6 +21,19 @@ def initialize(attributes = {})
   validates_with WorkflowStepsValidator
   validates_with WorkflowFiltersValidator
 
+  def call(user)
+    case
```

Comment:
Here we will implement the filters defined in [our trello](https://trello.com/c/BLeVSxhC/1751-implement-branches-filter), under the complete control of the workflow.